server/graphql/types/src/types/invoice_query.rs

```diff
@@ -451,6 +451,12 @@ impl InvoiceNode {
     pub async fn insurance_discount_percentage(&self) -> &Option<f64> {
         &self.row().insurance_discount_percentage
     }
+
+    pub async fn expected_delivery_datetime(&self) -> Option<DateTime<Utc>> {
```

Comment:
I think we should use 'date' for this, including in translations, otherwise we going from date to datetime and back to date in a few places

Reply:
Will need change in OG to get field to be date too? 👀  or just save as 0:00:00